replace_mlrun.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+# Copyright 2018 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+import os
+import subprocess
+from typing import List, Union
+
+
+class MlRunReplacer(object):
+    def __init__(self, args):
+        self._verbose = args.verbose
+
+    def _exec_cmd(self, cmd: List[str]) -> str:
+        if self._verbose:
+            print(" ".join(cmd))
+        return subprocess.run(
+            cmd,
+            capture_output=True,
+            text=True,
+            check=True,
+        ).stdout
+
+    def _exec_silent_cmd(self, cmd: List[str]):
+        if self._verbose:
+            print(" ".join(cmd))
+        subprocess.run(cmd, stdout=subprocess.DEVNULL, check=True)
+
+    def _get_mlrun_api_pod(self) -> Union[str, None]:
+        print("Get mlrun-api pod name")
+        cmd = ["kubectl", "-n", "default-tenant", "get", "pod"]
+
+        for line in self._exec_cmd(cmd).splitlines()[1:]:
+            if "mlrun-api-chief" in line:
+                return line.split()[0]
+
+        return None
```

Comment:
It's redundant to `return None` at the end of a function (but I guess it doesn't hurt either).